### PR TITLE
Update to latest Weave.

### PR DESCRIPTION
**Pull request by Syz66**
1. Fixes the mod for Weave Loader 0.2.4
2. Updated Gradle & Kotlin (changes nothing pretty much)
3. Changed license to GPL3 (due to weave being under gpl3)
4. Changed LCQT2 to LCQT (due to lcqt2 being c&ded)

### CODE REVIEW

**Review by Monmcgt — APPROVED** [review]

File: LICENSE.md
Comment (770grappenmaker) [inline LICENSE.md:1]: Weave Optifine URL is not a derivative work of Weave and should not have its license changed.

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Right now, it also assumes Minecraft 1.8.9, but does not assume a specific clien
 
 ### Installing
 1. Download [Weave Loader](https://github.com/Weave-MC/Weave-Loader/releases).
-2. Place `weave-optifine-url-0.1.jar` in `.weave/mods`.
-3. Launch Weave using a tool of choice, like [Weave Manager](https://github.com/exejar/Weave-Manager) or [LCQT2](https://github.com/nilsen84/lcqt2).
+2. Place `weave-optifine-url-0.2.jar` in `.weave/mods`.
+3. Launch Weave using a tool of choice, like [Weave Manager](https://github.com/exejar/Weave-Manager) or [LCQT](https://github.com/youded-byte/lunar-client-qt).
```
Comment (770grappenmaker) [inline README.md]: LCQT should just be removed and not replaced by a different version I suppose.

File: build.gradle.kts [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 dependencies {
-    compileOnly("com.github.weave-mc:weave-loader:v0.2.3")
+    compileOnly("com.github.weave-mc:weave-loader:v0.2.4")
 }
 
-kotlin { jvmToolchain(8) }
-minecraft.version("1.8.9")
\ No newline at end of file
+kotlin {
+    jvmToolchain(8)
```
Comment (770grappenmaker) [inline build.gradle.kts:21]: Beautiful formatting

File: src/main/kotlin/com/grappenmaker/optifineweave/WeaveOptifineURL.kt [hunk 3]
```diff
@@ -85,9 +86,9 @@ class CapeHook : Hook("net/optifine/player/CapeUtils") {
 }
 
 @Suppress("unused")
-class LunarBridgeHook : Hook("*") {
+class LunarBridgeHook : Hook() {
     override fun transform(node: ClassNode, cfg: AssemblerConfig) {
-        if (!node.name.startsWith("com/moonsworth/")) return
+        if (!node.name.startsWith("com/moonsworth")) return
```
Comment (770grappenmaker) [inline src/main/kotlin/com/grappenmaker/optifineweave/WeaveOptifineURL.kt:91]: Why

**Review by 770grappenmaker — CHANGES_REQUESTED** [review]

File: LICENSE.md
Comment (Syz66) [reply LICENSE.md:1]: Hm, I understand; I reverted the license change.

File: README.md (hunk 1 above)
Comment (Syz66) [reply README.md]: Alright, I removed LCQT.

File: build.gradle.kts (hunk 2 above)
Comment (Syz66) [reply build.gradle.kts:21]: I know right

File: src/main/kotlin/com/grappenmaker/optifineweave/WeaveOptifineURL.kt (hunk 3 above)
Comment (Syz66) [reply src/main/kotlin/com/grappenmaker/optifineweave/WeaveOptifineURL.kt:91]: I was having issues with the hook (I misunderstood something Nilsen said), and I must've removed that while messing around and forgot to re-add it. It doesn't change the functionality of the hook so it's fine to stay like that.

**Comment by Syz66** [comment]
Alright, I resolved all the issues you had. Feel free to look over it again.

**Review by 770grappenmaker — APPROVED** [review]

**Pull request merged**